std_msgs/Header header       # Standard ROS message header
BoundingBox[] bounding_boxes # Array of bounding boxes
